assets/js/DraggableContainer.js

```diff
@@ -0,0 +1,33 @@
+// Pasted from react-data-grid according to this comment:
```

Comment:
You shouldn't need to add this file -- it appears to be installable as part of react-data-grid-addons https://github.com/adazzle/react-data-grid/tree/master/packages/react-data-grid-addons